packages/manager/modules/pci/src/projects/project/instances/instances.controller.js

```diff
@@ -6,13 +6,15 @@ export default class CloudProjectComputeInfrastructureListCtrl {
   /* @ngInject */
   constructor(
     $q,
+    $state,
     $translate,
     CucCloudMessage,
     CucRegionService,
     ovhUserPref,
     PciProjectsProjectInstanceService,
   ) {
     this.$q = $q;
+    this.$state = $state;
```

Comment:
Please avois injecting `$state` in controller and prefer go through `resolve` and component `bindings`